src/compiler/parser.ts

```diff
@@ -5727,6 +5727,9 @@ namespace ts {
             function parseJSDocParameter(): ParameterDeclaration {
                 const parameter = <ParameterDeclaration>createNode(SyntaxKind.Parameter);
                 parameter.type = parseJSDocType();
+                if (parseOptional(SyntaxKind.EqualsToken)) {
+                    parameter.questionToken = createNode(SyntaxKind.QuestionToken);

```

Comment:
Shouldn't this be what we consider a "synthetic" node? That usually uses an `start` of `-1` or something.